azext_iot/central/_help.py

```diff
@@ -1752,3 +1754,257 @@ def _load_central_edge_help():
             --device-id {deviceId}
             --children-ids {child_1} {child_2}
     """
+
+
+def _load_central_enrollment_group_help():
+    helps[
+        "iot central enrollment-group"
+    ] = """
+          type: group
+          short-summary: Manage and configure IoT Central enrollment group
+      """
+
+    helps[
+        "iot central enrollment-group list"
+    ] = """
+    type: command
+    short-summary: Get the list of enrollment groups in an application
+    examples:
+      - name: Get the list of enrollment groups in an application
+        text: >
+          az iot central enrollment-group list
+          --app-id {appid}
+    """
+
+    helps[
+        "iot central enrollment-group show"
+    ] = """
+    type: command
+    short-summary: Get details about an enrollment group by ID
+    examples:
+      - name: Get details about an enrollment group by ID
+        text: >
+          az iot central enrollment-group show
+          --app-id {appid}
+          --id {enrollmentGroupId}
+    """
+
+    helps[
+        "iot central enrollment-group delete"
+    ] = """
+    type: command
+    short-summary: Delete an enrollment group by ID
+    examples:
+      - name: Delete an enrollment group by ID
+        text: >
+          az iot central enrollment-group delete
+          --app-id {appid}
+          --id {enrollmentGroupId}
+    """
+
+    helps[
+        "iot central enrollment-group create"
+    ] = """
+    type: command
+    short-summary: Create an enrollment group
+    examples:
+      - name: Create an enrollment group
```

Comment:
by default this makes a sym key enrollment group, no? Include that this will let the service generate the sym keys for you

Reply:
We don't have default behavior. Service will throw error: "message": "The provided request body contains the following errors: should have required property 'attestation' if customers didn't add attestation mechanism in the request.
And yes, only include 'symmetricKey' without any key will let service generate one for you.

Reply:
change this to use attestation sym key and have the help tell the users this is how to create an enrollment group with service generated keys. 

On that note, if a user just provides --at 'x509' then will we generate a cert for them or no? (we do this for `az iot hub device-identity create` but it may not be worth here)

Reply:
no we dont generate x509 for them,. it could be empty even they specify the attestation mechanism. Okay, then I use symmetric by default.